vendor/github.com/libopenstorage/openstorage/api/api.pb.go

```diff
@@ -7,7 +7,7 @@ package api
 import proto "github.com/golang/protobuf/proto"
 import fmt "fmt"
 import math "math"
-import google_protobuf "go.pedge.io/pb/go/google/protobuf"
+import google_protobuf "github.com/golang/protobuf/ptypes/timestamp"
```

Comment:
> Previously there were two different protobuf versions being used. I updated this vendor'ed code used by portworx to use the same protobuf version which others are using in openshift. Now we won't get the duplicate proto error.

Oh, ok.  Then this isn't a carry.  `<carry>` is a commit that updates code in a way that the upstream project rejected (or would reject if you know the project very well).  What you have done is a `bump`.  You should combine the code change the godep.json change into a single commit with a name like `bump(github.com/libopenstorage/openstorage): a53f5e5662367da02b95470980c5dbaadfe96c99`